src/faketypes.jl

```diff
@@ -110,7 +110,7 @@ isfakebottom(t::FakeTypeofBottom) = true
 function Base.:(==)(a::FakeTypeName, b::FakeTypeName)
     a.name == b.name && length(a.parameters) == length(b.parameters) || return false
     for i = 1:length(a.parameters)
-        a.parameters[i] === b.parameters[i] && return false
+        a.parameters[i] == b.parameters[i] || return false
     end
     return true
```

Comment:
Couldn't this whole thing just be written as `a.name == b.name && a.parameters == b.parameters`?

Reply:
Does comparison of vectors short circuit?

Reply:
I think so.